core/common/src/rpc/web/BentleyCloudRpcProtocol.ts

```diff
@@ -52,7 +52,7 @@ export abstract class BentleyCloudRpcProtocol extends WebAppRpcProtocol {
   /** Returns the operation specified by an OpenAPI-compatible URI path. */
   public override getOperationFromPath(path: string): SerializedRpcOperation {
     const url = new URL(path, "https://localhost/");
-    const components = url.pathname.split("/");
+    const components = url.pathname.split("/").filter((x) => x); // filter out empty segments
```

Comment:
With this change now https://dev-api.bentley.com/imodeljs/visualization/v3.0/mode/1/context/*/imodel/*/changeset/*/IModelReadRpcInterface-3.0.0-queryRows///// is allowed... I'm not sure whether this was intended.

Reply:
Yes, I don't see an issue with allowing consecutive slashes in any part of the URL. For example, APIM handles https://dev-api.bentley.com///imodel/rpc/mode/1/context/*/imodel/*/changeset/*/IModelReadRpcInterface-3.0.0-queryRows/ correctly when matching against `/imodel/rpc` pattern and I think that's good because this can happen when concatenating strings at the client side. I've never seen an API that intentionally allows empty URL segments, most of them just skip consecutive slashes or throw an unhelpful error like ours as of now.